docs/references/beamfits_memo.tex

```diff
@@ -0,0 +1,154 @@
+\url{http://pyuvdata.readthedocs.io/en/latest/uvbeam_parameters.html}.
+For examples of how to interact with this format using pyuvdata, please see the
+pyuvdata tutorial: \url{http://pyuvdata.readthedocs.io/en/latest/tutorial.html}.
+
+\section{Overview}
+There are two main types of \textit{beamfits} files, depending on whether the beams are 
+pixelized in HEALPix\footnote{\url{https://healpix.jpl.nasa.gov/}} or a regular pixel grid. There are also two subtypes of files for power 
+and E-field beams for each pixelization scheme. So there are four primary flavors of 
+\textit{beamfits} files which have small differences in format. There are also several optional 
+metadata components that may or may not be present in any given file. These variations are 
+described in detail in the following sections.
+
+\section{Primary HDU}
+The primary HDU of \textit{beamfits} files is an Image HDU containing the beam model.
+
+\subsection{Primary Header}
+The following are required keywords in the primary header of a  \textit{beamfits} file. For more detailed explanations 
+of what these keywords mean, see the descriptions on pyuvdata's ReadTheDocs uvbeam\_parameters page. 
+The UVBeam parameter corresponding to each keyword is noted in parentheses.
+As with all FITS files, HISTORY and COMMENT cards are allowed.
+
+\begin{itemize}
+\item{\textbf{BTYPE}: should be set to either `power' or `efield' to designate a power or E-field beam type. (beam\_type)}
+\item{\textbf{NORMSTD}: should be set to one of `physical', `peak' or `solid\_angle'. (data\_normalization)}
+\item{\textbf{COORDSYS}: should be set to one of `az\_za', `sin\_zenith' or `healpix'. (pixel\_coordinate\_system)}
+\item{\textbf{TELESCOP}: Telescope name. (telescope\_name)}
+\item{\textbf{FEED}: Feed name (used to distinguish between different physical feed types on the same telescope). (feed\_name)}
+\item{\textbf{FEEDVER}: Feed version (used to distinguish between different versions of the same feed on the same telescope). (feed\_version)}
+\item{\textbf{MODEL}: Model name  (used to distinguish between different models of the same hardware). (model\_name)}
+\item{\textbf{MODELVER}: Model version (used to distinguish between different versions of the same models of the same hardware). (model\_version)}
+\item{\textbf{FEEDLIST}: Only for E-field beams. The list of feeds orientations represented in the data. (feed\_array)}
+\item{\textbf{NSIDE}: Only for HEALPix beams. The NSIDE of the HEALPix map. (nside)}
+\item{\textbf{ORDERING}: Only for HEALPix beams. The ordering parameter of the HEALPix map.(ordering)}
+\end{itemize}
+
+\subsection{Beam Data}
+The axes of the data array in the primary HDU depends on the pixelization scheme.
+
+\subsubsection{HEALPix beam axes}
+For HEALPix beams, the data axes are: 
+\begin{enumerate}
+\item{\textbf{PIX\_IND}: index into the HEALPix pixel array (in the HPX\_INDS extension) This is needed because partial sky maps are supported so pixels may be missing.}
```

Comment:
Is this required for whole sky maps?

Reply:
I think it's fine as it is, but I believe it is a departure from standard Healpix Fits format. It may seem obvious, but I'd recommend noting explicitly in the comment that it is required even for full sky maps (not just that it's needed for partial sky maps).

Reply:
ok, will do.